Dokan/Dokan/Scenes/ProductDetails/Views/SliderCollectionCell/SliderCollectionViewCell.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  SliderCollectionViewCell.swift
+//  Dokan
+//
+//  Created by heba isaa on 19/07/2022.
+//
+
+import UIKit
+import UIDokan
+
+class SliderCollectionViewCell: UICollectionViewCell, IdentifiableView {
```

Comment:
Not needed since you already added a default implementation. 
```suggestion
class SliderCollectionViewCell: UICollectionViewCell {
```